sdk/ai/Azure.AI.Agents.Persistent/tests/Utilities/MemoryTraceExporter.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+using System.Collections.Generic;
+using OpenTelemetry;
+using OpenTelemetry.Trace;
+using System.Diagnostics;
+using Moq;
+
+namespace Azure.AI.Agents.Persistent.Tests.Utilities
+{
+    public class MemoryTraceExporter : BaseExporter<Activity>
```

Comment:
there is a built-in one - https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry.Exporter.InMemory/InMemoryExporter.cs